code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1132,13 +1134,7 @@
 
 /datum/ammo/xeno/toxin/New()
 	..()
```

Comment:
This is not really necessary. It will default to the parent proc.

Reply:
Do you want me to remove all of these blank entries?